content/docs/guides/assertions.md

```diff
@@ -11,27 +11,115 @@ An assertion query is written to find rows that violate one or more rules. If th
 
 ![Assertions](/static/images/assertions.png)
 
-## Example: Null checks
+## Auto generated assertions
+
+<div className="bp3-callout bp3-icon-info-sign" markdown="1">
+  Auto generated assertions are only supported from <code>@dataform/core</code> version <code>1.4.13</code>.
+</div>
+
+Dataform provides a convenient way to define assertions for tables, views and incremental tables as part of the `config` block of the table's SQLX file.
+
+Here's a complete example:
+
+`definitions/my_table.sqlx:`
+
+```js
+config {
+  type: "table",
+  assertions: {
+    uniqueKey: ["user_id"],
+    nonNull: ["user_id", "customer_id"],
+    rowConditions: [
+      'signup_date is null or signup_date > "2019-01-01"',
+      'email like "%@%.%"'
+    ]
+  }
+}
+select ...
+```
+
+For complete and up to date configuration options, view the [reference documentation for table assertions](/reference#ITableAssertions).
```

Comment:
nit: i would rm 'up to date', it implies we don't keep docs up to date, which i guess we don't, but let's not publish that fact :P